layouts/partials/fragments/contact.html

```diff
@@ -33,11 +33,18 @@ <h2>{{- . | markdownify -}}</h2>
     </div>
     <div class="row">
       <div class="col-12">
-        {{ if not $contact.posturl }}
-          <form class="contact" action="https://formspree.io/{{ $contact.email }}" method="POST" name="sentMessage" id="{{ $contact.formName | default "contactForm" }}">
-        {{ else }}
-          <form class="contact" action="{{ $contact.postURL }}" method="POST" name="sentMessage" id="{{ $contact.formName | default "contactForm" }}">
-        {{ end }}
+        <form class="contact" action="
+          {{- if not $contact.netlify -}}
+            {{- $postUrl := $contact.posturl | default (printf "https://formspree.io/%s" $contact.email) -}}
+            {{- printf $postUrl -}}
```

Comment:
What's the reason for the additional printf instead of just plain:
`{{ $contact.posturl | default (printf "https://formspree.io/%s" $contact.email) }}`
I'm fine with both, just wanted to get the reason behind that.

Reply:
It was left behind from the previous bug I wrote. Sorry.